account_statement_import_txt_xlsx/models/account_statement_import_sheet_mapping.py

```diff
@@ -66,9 +66,16 @@ class AccountStatementImportSheetMapping(models.Model):
     )
     amount_column = fields.Char(
         string="Amount column",
-        required=True,
```

Comment:
Please add an SQL constraint for having or this one, or the other 2. Some similar examples: https://github.com/odoo/odoo/blob/44ccecb994baa0f7bce4f5c39d6d136499e5be2e/addons/account/models/account_move.py#L3337

Reply:
Done